x-pack/legacy/plugins/uptime/index.ts

```diff
@@ -33,6 +34,13 @@ export const uptime = (kibana: any) =>
       home: ['plugins/uptime/register_feature'],
     },
     init(server: KibanaServer) {
-      initServerWithKibana(server);
+      const initializerContext = {} as PluginInitializerContext;
+      const { elasticsearch, xpack_main } = server.plugins;
```

Comment:
optional: you can also use the NP elasticsearch client by taking it from `server.newPlatform.setup.core.elasticsearch`